server/controllers/auth/clientSignIn.js

```diff
@@ -0,0 +1,33 @@
+const { compare } = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const connection = require('../../database/config/connection');
+
+const adminSignIn = async (req, res, next) => {
```

Comment:
```suggestion
const adminSignIn = async (req, res, next) => {
```
this is clientSignIn